WireAPI/Sources/WireAPI/Components/ResponseParser.swift

```diff
@@ -70,10 +70,10 @@ struct ResponseParser<Success> {
         return copy
     }
 
-    func failure(
+    func failure<E: Error>(
```

Comment:
I think it's the same and we have swiftformat configured to prefer the any. Or maybe it's going to change for this form with the swiftformat update, cc @samwyndham ?

Reply:
@netbe It would be the same if the original is `some`. But you are right. Our swift format config would change the following to be:

```swift
// Before format
func failure<E: Error>(error: E)

// After format
func failure(error: some Error)
```